nautobot/core/templates/inc/media.html

```diff
@@ -47,3 +47,4 @@
     <meta name="theme-color" content="#ffffff">
     <meta charset="UTF-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <meta name="robots" content="noindex, nofollow">
```

Comment:
Should this be conditional on the `PUBLISH_ROBOTS_TXT` setting?

Reply:
Done :)